tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -16,6 +16,12 @@
 
 using namespace std;
 
+static_assert(is_standard_layout_v<span<int>>);
+static_assert(is_standard_layout_v<span<int, 4>>);

```

Comment:
I suggest instead duplicating (and updating appropriately) the block on 90-97.

Reply:
Great idea :+1: